proto/message_api/v1/message_api.proto

```diff
@@ -7,50 +7,54 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 
 option go_package = "github.com/xmtp/proto/v3/go/message_api/v1";
 option java_package = "org.xmtp.proto.message.api.v1";
-
 option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
-    info: {
-        title: "MessageApi";
-        version: "1.0";
-    };
-};
+    info: { title: "MessageApi";
+version: "1.0";
+}
+;
+}
+;
 
 // RPC
 service MessageApi {
     // Publish messages to the network
-    rpc Publish (PublishRequest) returns (PublishResponse) {
+    rpc Publish(PublishRequest) returns (PublishResponse) {
         option (google.api.http) = {
-        post: "/message/v1/publish"
-        body: "*"
-    };
+            post: "/message/v1/publish"
+            body: "*"
+        };
     }
     // Subscribe to a stream of new envelopes matching a predicate
-    rpc Subscribe (SubscribeRequest) returns (stream Envelope) {
+    rpc Subscribe(SubscribeRequest) returns (stream Envelope) {
         option (google.api.http) = {
-        post: "/message/v1/subscribe"
-        body: "*"
-    };
+            post: "/message/v1/subscribe"
+            body: "*"
+        };
     }
+    // Subscribe to a stream of new envelopes and your subscription using
+    // bidirectional streaming
+    // protolint:disable:next RPC_REQUEST_STANDARD_NAME
+    rpc SubscribeV2(stream SubscribeRequest) returns (stream Envelope) {}
```

Comment:
Nitpick: How about `Subscribe2`, V2 makes it sound like there potentially will be V3 at some point, also might be confusing with relation to protocol versions.

Reply:
Works for me